### PR TITLE
HOLD till 2019: migrate altRoot to ts

### PR DESCRIPTION
**Pull request by CrispusDH**
It is just an example.
Should I change other files somewhere that depends on `altRoot`? @cnishina

### CODE REVIEW

**Comment by CrispusDH** [comment]
@cnishina any updates?

**Comment by CrispusDH** [comment]
kind of mess here, probably will create new branch

File: lib/taskRunner.ts [hunk 1]
```diff
@@ -7,12 +7,12 @@ import {Runner} from './runner';
...
-  specs: Array<string>;
-  capabilities: any;
+  taskId?: number;
+  specs?: Array<string>;
+  capabilities?: any;
   failedCount: number;
-  exitCode: number;
-  specResults: Array<any>;
+  exitCode?: number;
+  specResults?: Array<any>;
```
Comment (cnishina) [inline lib/taskRunner.ts]: I knew it. :smile:  (A comment from a previous PR that these should be optional)

Comment (cnishina) [reply lib/taskRunner.ts]: I also ran into this when working through my PRs.

**Pull request closed without merging**